hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -756,25 +756,61 @@
         ]
       }
     }
-    { multireg: {
-        name: "data_toggle_clear",
-        count: "NEndpoints"
-        cname: "Endpoint"
-        desc: "Clear the data toggle flag",
-        swaccess: "wo",
-        hwaccess: "hro",
-        hwqe: "true",
-        fields: [
-          {
-            bits: "0",
-            name: "clear",
-            desc: '''
-                  Writing 1 to this bit will clear the data toggle bit for this endpoint to Data0 in both IN and OUT directions.
-                  The register must no be written again within 200 ns.
-                  '''
-          }
-        ]
-      }
+    {
+      name: "out_data_toggle",
+      desc: "OUT Endpoints Data Toggles",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      hwqe: "true",
```

Comment:
Might want to just implement this outside `usbdev_reg_top` with the `hwext` tag.

```suggestion
      swaccess: "rw",
      hwaccess: "hrw",
      hwqe: "true",
      hwext: "true",
```

Reply:
Thank you. I have now returned to this PR, made that change, and it is much cleaner!
The registers are still specified as before; just the implementation has changed.